server/mysql_status.go

```diff
@@ -90,6 +98,30 @@ type MySQLReplicaStatus struct {
 	NetworkNamespace          string        `db:"Network_Namespace"`
 }
 
+type MySQLLastAppliedTransactionTimestamps struct {
+	OriginalCommitTimestamp string `db:"LAST_APPLIED_TRANSACTION_ORIGINAL_COMMIT_TIMESTAMP"`
+	EndApplyTimestamp       string `db:"LAST_APPLIED_TRANSACTION_END_APPLY_TIMESTAMP"`
```

Comment:
We may use [`mysql.NullTime`](https://pkg.go.dev/github.com/go-sql-driver/mysql#NullTime).